config.json

```diff
@@ -1348,6 +1348,18 @@
         "math"
       ]
     },
+    {
+      "slug": "knapsack",
+      "uuid": "ac179b77-98a5-4daf-9773-3d68b6cd8548",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Could you please define an exercise that will unlock this one? e.g. saddle-points

Reply:
Currently using saddle-points to unlock knapsack